.changeset/bright-signs-report.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-cloudflare": patch
+---
+
+Remove the Remix option since the framework is no longer recommended for new applications
```

Comment:
Maybe worth making it clear that it's Remix team recommend using React Router over Remix. Just in case people won't come across it being us not recommending remix anymore 😅

Reply:
Updated 0fe0bc39882ab9005ad231fce55e48868f26fb51 🙂 

Please let me know if this is what you had in mind 😄